rust/matches/src/lib.rs

```diff
@@ -768,6 +911,47 @@ pub struct CreateOrUpdateOracle<'info> {
     rent: Sysvar<'info, Rent>,
 }
 
+#[derive(Accounts)]
+pub struct MatchJoinNamespace<'info> {
+    #[account(mut)]
+    match_instance: Account<'info, Match>,
```

Comment:
`_instance` seems overly verbose. Just `match` seems sufficient

Reply:
you cant use match, thats a reserved word in rust

Reply:
Ah yes. Good call. `match_account` might be a better name then. Instance isn't very self describing.

Reply:
Okay, ill make this change but this is how it currently is in master, using `match_instance`.

Reply:
On second thought, MatchInstance is everywhere, I dont want to break the JS lib + all the integrations with match already so I'm inclined to keep it as `match_instance`